content/documentation/getting-started.md

```diff
@@ -39,109 +32,32 @@ Here are some of the most important tools to be aware of:
 
 If using kubernetes, then also review these tools:
 
-- [`kops`]({{< relref "tools/kops.md" >}})
 - [`helm`]({{< relref "tools/helm.md" >}})
 - [`helmfile`]({{< relref "tools/helmfile.md" >}})
 
 ## Kubernetes
 
-Kubernetes is a massive part of our solutions. Our Kubernetes documentation is geared towards leveraging [`kops`]({{< relref "tools/kops.md" >}}) by way of our `geodesic` strategy.
+Kubernetes is a massive part of our solutions. Our Kubernetes documentation is geared towards leveraging our [Terraform EKS modules](https://github.com/Cloudposse?q=terraform-aws-eks) and [`helmfile`]({{< relref "tools/helmfile.md" >}}).
 
 ### Helm
 
 Helm is central to how we deploy all services on kubernetes.
 
 * [helm]({{< relref "tools/helm.md" >}}) is essentially the package manager for Kubernetes (like `npm` for Node, `gem` for Ruby, and `rpm` for RHEL)
-* [helm charts](/helm-charts/) are how kubernetes resources are templatized using Go templates
-* [helm charts quickstart]({{< relref "helm-charts/quickstart.md" >}}) is our "cheatsheet" for getting started with Helm Charts.
-* [helm registries](/kubernetes-platform-services/chart-registry/) are used to store helm charts, which are essentially tarball artifacts.
-* [chartmuseum]({{< relref "kubernetes-platform-services/chart-registry/chartmuseum.md">}}) is deployed as the chart repository
+* [helm charts](https://helm.sh/docs/topics/charts/) are how kubernetes resources are templatized using Go templates
 * [helmfiles]({{< relref "tools/helmfile.md">}}) are used to define a distribution of helm charts. So if you want to install prometheus, grafana, nginx-ingress, kube-lego, etc, we use a `helmfile.yaml` to define how that's done.
-* [chamber]({{< relref "tools/chamber.md">}}) is used to manage secrets and provide them when provisioning with `helmfile`. It's also a big part of our overall story on [secrets management](/secrets-management/)
+* [chamber]({{< relref "tools/chamber.md">}}) is used to manage secrets and provide them when provisioning with `helmfile`. It's also a big part of our overall process around secrets management.
```

Comment:
We really don't use `chamber` any more since `helmfile` supports SSM natively or we use secrets already provisioned in the cluster.
```suggestion
```